.rubocop.yml

```diff
@@ -0,0 +1,45 @@
+# use the shared YaST defaults
+inherit_from:
+  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+# Redundant returns add legibility for developers used to other languages
+Style/RedundantReturn:
+  Enabled: false
+
+# Don't complain on definitions like this. We like question marks!
+# def some_boolean?; some_boolean; end
+TrivialAccessors:
+  ExactNameMatch: true
```

Comment:
it is wrong one, it is actually option https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L895 but I also use this, but its goal is to allow non exact match so something like
```
def name
  @intenal_name
end
```
AllowPredicates is about question mark at the end, so it is subset of exact match